plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -576,6 +576,9 @@ export class EventsProcessor {
 
         const elementsChain = elements && elements.length ? elementsToString(elements) : ''
 
+        // TODO: don't parse back and forth with json
+        const personInfo = await this.db.getPersonInfoThroughCacheFromDistinctId(teamId, distinctId)
```

Comment:
this func name is long (not as big of a deal) and a bit confusing - "_through_" cache "_from_" distinct id?

More like _from_ cache _by_ distinct id.

I think just `getCachedPersonData` does fine. The params explain to me _how_ it's done

Reply:
well if it's not in cache we'll get it from psql, so I might just name it `getPersonData` then